src/WebJobs.Script/Host/IHostFunctionMetadataProvider.cs

```diff
@@ -12,13 +12,13 @@ namespace Microsoft.Azure.WebJobs.Script
     /// <summary>
     /// Defines an interface for fetching function metadata from function.json files
     /// </summary>
-    internal interface IHostFunctionMetadataProvider
+    public interface IHostFunctionMetadataProvider
```

Comment:
Why is this no longer internal?

Reply:
Needed to make it public to use within a public class. WebfunctionManager was public already.

Reply:
Yes, the visibility on these classes wasn't managed very well. WebFunctionsManager should have been internal